api/discovery.capnp

```diff
@@ -0,0 +1,41 @@
+using Go = import "/go.capnp";
+
+@0xfcba4f486a351ac3;
+
+$Go.package("discovery");
```

Comment:
I've been thinking:  naming this package `discovery` is bound to cause confusion with [libp2p's discovery package](https://pkg.go.dev/github.com/libp2p/go-libp2p/core/discovery).

What if we named it `service` instead?